### PR TITLE
README.md: add ref to first MemGuard for Jailhouse

### PR DESCRIPTION
**Pull request by soujak**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,6 +27,10 @@ centers and universities:
 
 The extensions have been evaluated in multiple academic publications such as:
 
+> Marco Solieri, Tomasz Kloda, Marko Bertogna, Michal Sojka, Maxim Baryshnikov,
+> "D5. 3: Integrated schedulability analysis",
+> Deliverable of the HERCULES project, 2018
```
Comment (bastoni) [inline README.md:32]: @soujak Please provide a direct link and a reference to the pages. All the other reference can be found easily with a simple search and are self contained. This one cannot be easily accessed.

Comment (soujak) [reply README.md:32]: The project website disappeared, but we can provide the latest URL available from The Internet Archive.

Comment (soujak) [reply README.md:32]: And also the academic repository provided by UNIMORE.

Comment (bastoni) [reply README.md:32]: Thanks. I think the iris one from UNIMORE is sufficient. Can you please repush with signoff? It's missing.

**Comment by soujak** [comment]
Obsoleted by [README.md: add ref to first MemGuard for Jailhouse](https://github.com/Minervasys/jailhouse/pull/3).

**Pull request closed without merging**